tools/sotn-assets/assets/spritebanks/spritebanks.go

```diff
@@ -96,8 +96,8 @@ func buildSprites(fileName string, outputDir string) error {
 	if err := json.Unmarshal(data, &spritesBanks); err != nil {
 		return err
 	}
-	if len(spritesBanks.Indices) != banksCount {
-		return fmt.Errorf("the number of banks must be exactly %d, got %d", banksCount, len(spritesBanks.Banks))
+	if len(spritesBanks.Indices) != banksCount && len(spritesBanks.Indices) != banksCount + 1 {
```

Comment:
This is very much a hack around.
NO0 has 25 sprite banks, vs. the expected 24 `const banksCount = 24 // the number seems to be fixed`

I don't know if you have any good ideas for this @Xeeynamo?